vscode/test/buildTests.mjs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+//@ts-check
+
+import { dirname, join } from "node:path";
+import { fileURLToPath } from "node:url";
+import { build } from "esbuild";
+
+const thisDir = dirname(fileURLToPath(import.meta.url));
+
+/** @type {import("esbuild").BuildOptions} */
+const buildOptions = {
+  entryPoints: [
+    join(thisDir, "suites", "language-service", "index.ts"),
+    join(thisDir, "suites", "debugger", "index.ts"),
+  ],
+  outdir: join(thisDir, "out"),
+  bundle: true,
+  mainFields: ["browser", "module", "main"],
+  external: ["vscode"],
+  format: "cjs",
+  platform: "browser",
+  target: ["es2020"],
+  sourcemap: "linked",
+  //logLevel: "debug",
+  define: { "import.meta.url": "undefined" },
+};
+
+function buildTests() {
```

Comment:
Likewise, this top level function adds no value if it's not exported and just called once immediately after definition. Just use top-level statements.